src/utils/collection.ts

```diff
@@ -142,6 +142,30 @@ const checkConsentsFromValues = (publisherConsents: Consent[], consents: Record<
   return { customerConsents, consentsErrors };
 };
 
+const isNotEmptyStringEntry = <T>([, value]: [string, T]) => value !== '';
+
+const formatCrfEntry = <T>([name, value]: [string, T], _: number, collection: [string, T][]) => {
```

Comment:
Please avoid obscure abbreviations like "Crf". Prefer to just call it `formatRegisterFieldEntry`